modules/ROOT/pages/global-settings-configuration.adoc

```diff
@@ -55,18 +69,19 @@ Specifies defaults and general settings for the Mule application instance.
 *Default*: none
 |===
 
-=== Child Elements of <configuration...>
+== Child Elements of <configuration>
 
 [%header%autowidth.spread]
 |===
-|Name |Cardinality |Description
-|`dynamic-config-expiration` |0..1 |The default expiration settings for dynamic configurations, used by connectors. This can also be configured on connectors, in which case the connector configuration is used instead of this default.
-|`expression-language` |0..1 |Configuration of Mule Expression Language (MEL).
-|`cluster-config` |0..1 |Mule application cluster configuration.
+|Name |Description
```

Comment:
Consider using list format instead of a table for two-column <thing>+<description> texts, especially when the second column runs longer than a sentence.
See "definition or description lists (dlists)" and "tables" in the style ref. for further guidance.